### ISSUE TITLE
Incorrect token assignment on SDDR card 

### ISSUE THREAD

**Issue by repositony** [repositony/vscode_mcnp#1]
The `contrib` variable on the `SDDR` card is highlighted as a `keyword.mcnp` when it should be a `variable.mcnp`. The constants that follow are working as intended.

**Comment by repositony** [comment]
Simple fix, just change the `name` 
```yaml
contrib:
    match: (?i)(contrib)[\s\=]*(zaid|cell|voxel)?[\s\n]
    captures:
      "1":
        name: variable.mcnp
      "2":
        name: constant.language.mcnp  
```

**Comment by repositony** [comment]
Fixed as part of commit #59f2795
